src/Microsoft.Health.Dicom.Api/Controllers/RetrieveController.cs

```diff
@@ -47,6 +48,7 @@ public RetrieveController(IMediator mediator, ILogger<RetrieveController> logger
             _logger = logger;
         }
 
+        [Produces(KnownContentTypes.MultipartRelated + TypeSeperator + KnownContentTypes.ApplicationDicom)]
```

Comment:
Should we just create a `KnownContentTypes` for this combination?

Reply:
Just made them only return multipart